pdfminer/pdfinterp.py

```diff
@@ -141,10 +143,16 @@ class PDFResourceManager:
     allocated multiple times.
     """
 
-    def __init__(self, caching=True):
+    def __init__(self, caching: bool = True,
```

Comment:
Where are PDFResourceManager instances created?

Reply:
These are created in Arnold when PDFParser is initialized.